src/new-client/testing/testEnvironment.ts

```diff
@@ -5,8 +5,12 @@ dotenv.config({ path: path.resolve(__dirname, 'playwright.env') });
 
 const environment = {
   BASE_URL: process.env.BASE_URL ?? '',
-  TEST_USER_USERNAME: process.env.TEST_USER_USERNAME ?? '',
-  TEST_USER_PASSWORD: process.env.TEST_USER_PASSWORD ?? '',
+  TEST_USER_1_USERNAME: process.env.TEST_USER_1_USERNAME ?? '',
+  TEST_USER_1_PASSWORD: process.env.TEST_USER_1_PASSWORD ?? '',
+  TEST_USER_2_USERNAME: process.env.TEST_USER_2_USERNAME ?? '',
+  TEST_USER_2_PASSWORD: process.env.TEST_USER_2_PASSWORD ?? '',
+  TEST_USER_3_USERNAME: process.env.TEST_USER_3_USERNAME ?? '',
+  TEST_USER_3_PASSWORD: process.env.TEST_USER_3_PASSWORD ?? '',
```

Comment:
I intended to wrap these up a bit to avoid so much noddy repetition - I intend to do it alongside introducing Page Object Model and other Playwright cleanups though